lib/commands/web.js

```diff
@@ -387,13 +387,19 @@ helpers.useAtomsElement = function (el) {
 helpers.convertElementsForAtoms = function (args = []) {
   let newArgs = [];
   for (let arg of args) {
-    if (!_.isNull(arg) && !_.isUndefined(arg.ELEMENT)) {
-      let atomsElement = this.getAtomsElement(arg.ELEMENT);
-      if (atomsElement === null) {
-        throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${arg.ELEMENT}'`);
+    let hasElementKey = false;
+    for (let elementKey of ['ELEMENT', 'element-6066-11e4-a52e-4f735466cecf']) {
+      if (!_.isNull(arg) && !_.isUndefined(arg[elementKey])) {
+        hasElementKey = true;
+        let atomsElement = this.getAtomsElement(arg[elementKey]);
+        if (atomsElement === null) {
+          throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${arg.ELEMENT}'`);
+        }
+        newArgs.push(atomsElement);
       }
-      newArgs.push(atomsElement);
-    } else {
+    }
+
+    if (!hasElementKey) {
```

Comment:
why do we need hasElementKey flag if we can simply put it into `else`?

Reply:
https://www.w3schools.com/js/js_break.asp

Reply:
It shouldn't have been in the for-loop.